Assets/Scripts/Utility/RMBLayout.cs

```diff
@@ -36,6 +40,11 @@ public static class RMBLayout
         public const uint CityGateClosedModelID = 447;
         public const uint BulletinBoardModelID = 41739;
 
+        private static int maxLocationCacheSize = 12;
```

Comment:
How did you arrive at 12 for the cache size?

Reply:
Indeed, I chose this number since it covers all locations currently loaded at a given position on the world map, as well as a dungeon and locations near it on another point. So if player gets a quest that sends them to a dungeon, and then returns the questgiver town, no loading required.